packages/react/src/primitives/types/card.ts

```diff
@@ -1,6 +1,6 @@
 import { ViewProps } from './view';
 
-export type CardVariations = 'outlined' | 'elevated';
+export type CardVariations = 'outlined' | 'elevated' | 'default';
```

Comment:
Why we need a `default`? Shouldn't `undefined` means default like other primitives? I didn't see we use default modifier in CSS